### PR TITLE
(docs) Elements code tabs

### PR DESCRIPTION
**Pull request by connor-leech**
These docs are 1:1 with the package readmes, bringing to the site for better visibility.

Closes https://github.com/carbon-design-system/carbon-website/issues/73.

### CODE REVIEW

File: src/pages/guidelines/2x-grid/code.mdx [hunk 1]
```diff
@@ -16,9 +22,32 @@ tabs: ["Basics", "Implementation"]
 
 </AnchorLinks>
 
-## Under the hood
+## Resources
```
Comment (joshblack) [inline src/pages/guidelines/2x-grid/code.mdx:25]: Would we also want to link to the React storybook here? https://react.carbondesignsystem.com/?path=/story/grid--auto-columns

File: src/pages/guidelines/themes/code.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+Use the themes package to customize your product while maintaining consistency.
+
+</PageDescription>
+
+## Resources
+
+<Row className="resource-card-group">
+  <Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Carbon layout package"
```
Comment (joshblack) [inline src/pages/guidelines/themes/code.mdx:19]: Should this be the themes package? 👀

Comment (connor-leech) [reply src/pages/guidelines/themes/code.mdx:19]: Yes, good catch!

File: src/pages/guidelines/2x-grid/code.mdx (hunk 1 above)
Comment (connor-leech) [reply src/pages/guidelines/2x-grid/code.mdx:25]: 👍

File: src/pages/guidelines/icons/contribute.mdx [hunk 3]
```diff
@@ -2,7 +2,7 @@
 label: Icons are visual symbols used to represent ideas, objects, or actions. They communicate messages at a glance, afford interactivity, and draw attention to important information.
 title: Icons
 description: Icons are visual symbols used to represent ideas, objects, or actions. They communicate messages at a glance, afford interactivity, and draw attention to important information.
-tabs: ["Library", "Usage", "Contribute"]
+tabs: ["Library", "Usage", "Code,"]
```
[the remark this answers is not in the document]
Comment (connor-leech) [reply src/pages/guidelines/icons/contribute.mdx:5]: 👍

File: src/pages/guidelines/icons/library.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ const frontmatter = {
   title: 'Icons',
   description:
     'Icons are visual symbols used to represent ideas, objects, or actions. They communicate messages at a glance, afford interactivity, and draw attention to important information.',
-  tabs: ['Library', 'Usage', 'Contribute'],
+  tabs: ['Library', 'Usage', 'Code'],
```
[the remark this answers is not in the document]
Comment (connor-leech) [reply src/pages/guidelines/icons/library.js:11]: 👍

**Pull request merged**